packages/components/src/local/molecules/sortable-list/index.tsx

```diff
@@ -66,18 +66,7 @@ export const SortableList: React.FC<PropTypes> = React.forwardRef(({
   }
 
   const handleRemove = (key: number): void => {
-    const newItems = [...items]
```

Comment:
@lilitkarapetyan - I'm testing this branch. We need this removed code - for cases other than the roles listing (the platform-types admin page)